app/scripts/app.slickCarousel.ts

```diff
@@ -0,0 +1,40 @@
+// Import slick-carousel from node_modules 
+// Loading from /scripts because https://github.com/kenwheeler/slick/pull/3307
+import "slick-carousel";
+
+// Testimonialz Slick slider
+export function testimonialsSlider() {
+	const slickSlider = $('.slider-center');
```

Comment:
`import "slick-carousel";`

but 

` $('.slider-center');`

Please use `'` and `"` consistently. In JS `"` used most of the time.